spec.html

```diff
@@ -8467,6 +8469,9 @@ <h1>[[GetOwnProperty]] (_P_)</h1>
           <li>
             A property cannot be reported as non-configurable, if it does not exist as an own property of the target object or if it exists as a configurable own property of the target object.
           </li>
+          <li>
+            A property cannot be reported as both non-configurable and non-writable, unless it exists as a non-configurable, non-writable own property of the target object.
```

Comment:
This sentence uses “unless it exists as an X own”, while the others in this section use “if it does not exist as an own, or if it exists as an X own” - i like your wording better, so maybe we should update them all to use this form? That’s also fine to do in a followup, but in that case, this should probably follow the form of the others in the interim?